src/en/space-station-14/departments/science/proposals/scientific-research-proposal.md

```diff
@@ -0,0 +1,35 @@
+# Scientific research proposal
+
+| Designers     | Implemented | GitHub Links |
+|---------------|-------------|--------------|
+| chromiumboy   | :x: No      | TBD          |
+
+## Problem statement
+At present the science department faces several key gameplay issues when it comes to unlocking items through scientific research. These include (but are not limited to):
+-	A large number of research projects that can be considered ‘filler’. These projects unlock either very niche or very common place items on the station; a choice between unlocking magboots, hydroponic trays, or buildable research disk terminals is not compelling for the player. 
+-	Due to the high proportion of filler projects at lower research tiers, typically one project will be seen as the ‘optimal’ choice to research (generally either in the industrial or experiment disciplines), leading to many niche items being ignored in the early to mid stages of the shift when they would be most useful, as well as a sense of sameness in technology unlocking despite the research projects being presented at random.
+-	Many research items seem to be under costed with respect to the current ease of points generation. This leads to shifts where all research projects can be readily completed, particularly during longer than normal shifts and / or shifts that have a large research team.
+-	Once all research projects have been completed, there is no longer any compelling reason to conduct any further research.
+
+## Proposed changes
```

Comment:
I'd propose using this 

The changes to scientific research outlined in this proposal have been devised in an attempt to alleviate the issues listed above. These changes include:
-	Making all low-tier / filler researchable items available to print at round start. These include items such as proximity sensors, TV circuit boards, and surveillance camera equipment.
-	Making the more desirable mid-tier researchable items (e.g., advanced cleaning, lasers, and high-capacity batteries) also available to print at round start, but with the caveat that science points are required as part of their construction. Lore-wise, this can be explained as the technology already being widely known, however, you need to purchase technology licenses from NT in order to print them (and NT is mostly interested in data-sets that RD server produces!).
-	Scientists can print technology licenses from the tech disk terminal, exchanging their research points for discipline specific licenses (e.g., a single civilian technology license might cost 150 research points, a single industrial technology license 250, and a single arsenal technology license 350).
-	These licenses can then be fed into (or ejected from) an appropriate lathe just like any other resource material (e.g., arsenal technology licenses can be inserted into security and ammo fabs, civilian technology licenses into medical fabs, and experimental technology licenses into protolathes). That means that each department will get its own kind of fab - med and sec have it currently, engi, cargo and service will get new ones, and sci will get fancy recolored one.
-	Licensed items require a number of licenses that correspond to their relative power / usefulness (e.g., a laser carbine might require five licenses to print, in addition to their other base materials, while a box of incendiary ammunition might only need one license). 
-	The highest tier / most powerful technologies (e.g., bluespace backpacks, speed boots, teleporter pads; perhaps a dozen technologies all up) would still need to be researched the traditional way, requiring an expenditure of research points at the R&D computer to unlock (in addition to the license cost to print them afterward; unfortunately, NT owns all patentable technology produced by their research staff). However, each of these research projects now requires a vast amount of research points to unlock, in the range of hundreds of thousands. The ultimate goal is that unlocking a new technology should be a relatively rare occurrence and a nice trading barging when trying to get something from other department. Maybe we can expected in the timeline of one shift,to have only two or three unlocks per shift being common. Major new research cost will also motivate sci to give away licenses for crafting easier - it doens't really matter much if u give away 1000-2000 sci points to janitor, if your aim is to collect, for example, 80 000 - its not comparable, it won't hurt you that much in the long run.


```mermaid
flowchart LR
	A1 -- Fixed by --> B1
	A2 -- Fixed by --> B1
	A3 -- Fixed by --> B1
	A4 -- Also fuels --> C2
	A5 -- Fixed by --> B2
	A6 -- Fixed by --> B2
	B1 -- Leads to --> C1
	B1 -- Leads to --> C2
	B2 -- Leads to --> C4
	C1 -- Fixed by --> D1
	C2 -- Fixed by --> D2
	C3 -- Fixed by --> D1
	C4 -- Fixed by --> D1
	
	subgraph Problems[baseline problems]
		A1['filler' tech <br>doesn't feels involving]
		A2['non-target' t1/t2 disciplines <br>rarely being researched <br >in early game, when <br> they are most useful]
		A3['filler' tech is obstructing <br> view of 'useful tech' ]
		A4[Important researches <br>have inadequate price]
		A5[Sci unwilling <br>to waste a lot <br>of time on reception]
		A6[Sci unwilling <br>to share hard-earned <br>hard-recoverable - for them <br>- resources]
	end
	subgraph proposed [proposed solutions]
		B1[Move 'filler' tech <br>to be available roundstart]
		B2[Each department <br>gets own 'protolathe' <br>with recepie restrictions]
	end
	subgraph intermediate[intermediate problems]
		C1[Diminishing role <br> of sci department]
		C2[Less 'content' <br>for science department]
		C3[Almost nothing to do after <br>most desired tech <br> was researched]
		C4[Why bother giving<br> something or help sci?]
	end
	subgraph final[final fixes]
		D1[Add some currency, <br>made from science points <br> to be used in most <br>of printed tech recepies]
		D2[Rise tech research <br>base costs and use dynamic <br> research cost mechanic]
	end

```

Reply:
i will DM graph code to you in discord

Reply:
Thanks, that looks really cool